addons/resource.language.en_gb/resources/strings.po

```diff
@@ -17580,4 +17580,15 @@ msgstr ""
 #: system/settings/settings.xml
 msgctxt "#38017"
 msgid "Region A - Americas, East Asia and Southeast Asia. Region B - Africa, Middle East, Southwest Asia, Europe, Australia, New Zealand. Region C - Central Asia, mainland China, Mongolia, South Asia, Belarus, Russia, Ukraine, Kazakhstan."
+
+#. Setting #38118 "Settings -> System -> Audio output -> Support 8 channel DTS-HD audio decoding"
+#: system/settings/settings.xml
+msgctxt "#38118"
+msgid "Support 8 channel DTS-HD audio decoding"
+msgstr ""
+
+#. Description of setting #38118 Settings -> System -> Audio output -> Support 8 channel DTS-HD audio decoding
+#: system/settings/settings.xml
+msgctxt "#38119"
+msgid "Enables decoding of high quality DTS-HD audio streams. Note: This increases CPU load and is only available when DTS and DTS-HD audio passthrough are disabled."
```

Comment:
I did not agree on having this silly setting in global. This is just another step towards even more mess with settings. There is absolutely no reason for the majority of devices to have this disabled. For the other either it works or not so this can be set as an appliance for this low end devices.
In addition pass though is an option to PCM audio. Following this logic giving pass through priority is wrong.